templates/cloudformation/aws_agent_with_telemetry_collector_and_vpc.yaml

```diff
@@ -0,0 +1,136 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
this parent stack is an example that deploys the MC Agent with the Telemetry Collector and creates a new basic VPC in AWS